packages/@react-spectrum/tooltip/src/TooltipTrigger.tsx

```diff
@@ -30,8 +31,19 @@ function TooltipTrigger(props: SpectrumTooltipTriggerProps) {
     offset = DEFAULT_OFFSET
   } = props;
 
+  let tooltipRef = useRef();
   let [trigger, tooltip] = React.Children.toArray(children);
 
+  // @ts-ignore
+  let tooltipWithRef = React.cloneElement(tooltip, {ref: node => {
+    tooltipRef.current = node;
+    // @ts-ignore
+    if (tooltip.ref) {
+      // @ts-ignore
+      tooltip.ref.current = node;
+    }
+  }});
```

Comment:
I couldn't find a better way to do this

Reply:
I think this will conflict with a user provided ref on a tooltip?
I think we can use context and resolve them to the same ref during a useEffect

Reply:
i'm guessing merging them in a useEffect is too late?

Reply:
What do you have in mind?

Reply:
we have some ref merging logic around the repo that takes place in a useEffect, trying to find some examples
it uses context usually though if i recall correctly

Reply:
here's the one i was thinking of https://github.com/adobe/react-spectrum/blob/main/packages/@react-aria/interactions/src/DOMPropsContext.ts#L37
I've no idea if it'll help you out

Reply:
I don't think so, it could still be the case that the tooltip doesn't have a ref yet and I still need to do `cloneElement` then...